### PR TITLE
Creator names black

### PR DESCRIPTION
**Pull request by TillSchlemmermeier**
removed creator_name property to contents-model

added contents.update in controller to get the creator_name on content-creation

### CODE REVIEW

File: app/models/person.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class Person
   include Neo4j::ActiveNode
+  property :name, type: String
```
Comment (malachewhiz) [inline app/models/person.rb]: Remove this property

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #README
 
 This README would normally document whatever steps are necessary to get the
-application up and running.
+application up and running.s
```
Comment (lazarDevelopment) [inline README.md]: remove the 's'

**Comment by malachewhiz** [comment]
specs?

**Comment by TillSchlemmermeier** [comment]
specs done

**Comment by lazarDevelopment** [comment]
ok, please merge

**Pull request merged**